src/main/resources/templates/java/gradle_gradle/exercise/git.ignore.file

```diff
@@ -0,0 +1,94 @@
+assignment/src/
+assignment/resources/
+
+*.class
+.idea
+bin
+build
+/target/
+/target/*
+*.jar
+*.war
+*.ear
+
+#################
+# Eclipse .gitignore
+# Source: https://github.com/github/gitignore/blob/master/Global/Eclipse.gitignore
+
+.metadata
+bin/
+tmp/
+*.tmp
+*.bak
+*.swp
+*~.nib
+local.properties
+.settings/
+.loadpath
+.recommenders
+
+# Eclipse Core
+.project
+
+# External tool builders
+.externalToolBuilders/
+
+# Locally stored "Eclipse launch configurations"
+*.launch
+
+# PyDev specific (Python IDE for Eclipse)
+*.pydevproject
+
+# CDT-specific (C/C++ Development Tooling)
+.cproject
+
+# JDT-specific (Eclipse Java Development Tools)
+.classpath
+
+# Java annotation processor (APT)
+.factorypath
+
+# PDT-specific (PHP Development Tools)
+.buildpath
+
+# sbteclipse plugin
+.target
+
+# Tern plugin
+.tern-project
+
+# TeXlipse plugin
+.texlipse
+
+# STS (Spring Tool Suite)
+.springBeans
+
+# Code Recommenders
+.recommenders/
+
+#################
+# Netbeans .gitignore
+# Source: https://github.com/github/gitignore/blob/master/Global/NetBeans.gitignore
+
+nbproject/private/
+build/
+nbbuild/
+dist/
+nbdist/
+.nb-gradle/
+
+
+#################
+# IntelliJ .gitignore
+# Source: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems
+.idea
+*.iws
+*.iml
+out
+
+################
+# Gradle
+.gradle
+gradlew.bat
```

Comment:
Is there any specific reason why you exclude all Gradle wrapper files except `gradle/wrapper/gradle-wrapper.jar` here? (And in the other git ignore files as well)
Usually, the Gradle wrapper is shipped together with the repository. It would also make sense IMO to put the gradle wrapper in the template and solution repos, as this would make the Execution of the code easier.

Unless Stephan has anything against it I think we should include the gradle wrapper in the template.

Reply:
This is a relict and I will change to exclude all wrapper files.
We will not use the wrapper but the `gradle` command instead directly. This has to do with the way the builds are executed on Bamboo in the corresponding Docker container.
In case the wrapper is required, e.g. to run tests with a specific Java/Gradle version, the .gitignore can still be edited and the build plan can be changed accordingly.